src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdTypeDataProvider.java

```diff
@@ -16,7 +16,7 @@ private AdTypeDataProvider() {
     return new Object[][]{
         {
             "adtest", "SyntheticTests/Async/Success",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "adtest" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)